examples/.env.example

```diff
@@ -0,0 +1,3 @@
+IMX_APP_USER_WALLET_PRIVATE_KEY=
```

Comment:
Do we want to call this `DEPLOYER_WALLET_PRIVATE_KEY`

Reply:
No because only collection and mint would call this deployer but other flows like trade and transfer wouldn't make sense to call this a deployer.

Reply:
Renamed to `PRIVATE_KEY`
[b2a9ab9](https://github.com/immutable/imx-core-sdk/pull/195/commits/b2a9ab9e2a9ebbeefea8344b89fbe6807e754263)